ruma-identifiers/src/room_version_id.rs

```diff
@@ -144,7 +144,7 @@ impl RoomVersionId {
     /// Whether or not this is a version 6 room.
     #[deprecated = "compare to RoomVersionId::Version6 instead"]
     pub fn is_version_6(&self) -> bool {
-        matches!(self, Self::Version5)
+        matches!(self, Self::Version6)
```

Comment:
Ouch. Good thing it's deprecated.

Reply:
Oops, I kind of copy/pasted everything with a '5' in it, I'll fix that :)

Reply:
You didn't do anything wrong, you fixed multiple bugs at once! I was saying it's good the one bug was in deprecated code.

Reply:
I still believe I should remove the deprecated functions I added like `is_version_6()`, and fix the tests like this...
```diff
-        assert!(!custom.is_version_6());
+        assert_ne!(custom, RoomVersionId::Version6);
```

Reply:
`is_version_6` already existed prior to your PR. Using `assert_ne!` in the tests sounds like a good idea though.

Reply:
Now that I think about it though, we already a breaking change in ruma-identifiers on master anyway, feel free to also remove *all* the deprecated stuff from ruma-identifiers in another PR :)